package.json

```diff
@@ -65,14 +65,14 @@
     "@bigtest/mirage": "^0.0.1",
     "@bigtest/mocha": "^0.5.0",
     "@bigtest/react": "^0.1.2",
-    "@folio/eslint-config-stripes": "^3.2.1",
+    "@folio/eslint-config-stripes": "^5.0.0",
     "@folio/stripes": "^2.0.0",
     "@folio/stripes-cli": "^1.4.0",
     "@folio/stripes-core": "^3.0.1",
     "babel-eslint": "^9.0.0",
```

Comment:
Please consider updating of babel-eslint according to the [comment here](https://github.com/folio-org/ui-search/pull/200#discussion_r372879828) in order to prevent further misleading eslint complaints?

Reply:
Done